tools/RAiDER/aria/prepFromARIA.py

```diff
@@ -0,0 +1,212 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Author: Jeremy Maurer, Brett Buzzanga
+# Copyright 2022, by the California Institute of Technology. ALL RIGHTS
+# RESERVED. United States Government Sponsorship acknowledged.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+import os
+from datetime import datetime
+import numpy as np
+import xarray as xr
+import rasterio
+import yaml
+import RAiDER
+from RAiDER.utilFcns import rio_open, writeArrayToRaster
+from RAiDER.logger import logger
+
+""" Should be refactored into a class that takes filename as input """
+## ---------------------------------------------------- Prepare Input from GUNW
+def makeLatLonGrid(f:str, out_dir:str):
+# def makeLatLonGrid(f:str, reg, out_dir):
+    ds0  = xr.open_dataset(f, group='science/grids/data')
+
+    Lat, Lon  = np.meshgrid(ds0.latitude.data, ds0.longitude.data)
```

Comment:
See my comment on the heights; related also to the lonlat. 
- Grids for the tropo needs to be fixed for all products around the world such we can build a vrt of multiple cubes (GUNW products)

Reply:
I think we should move this discussion to #441 and just leave this branch as implementing the CLI changes

Reply:
Looks like answer to my comment above is the cubes. We may then want to define the global grid similar to how height levels are defined for weather models.